src/protocol/DocumentServer.js

```diff
@@ -37,15 +39,16 @@ define('polymer-designer/protocol/DocumentServer', [
       this.nodes = new Map();
       this.nextId = 1;
 
+      connection.on('command', this._onCommand.bind(this));
+      connection.on('getCaretPosition', this.getCaretPosition.bind(this));
```

Comment:
could be... there isn't actually a lot of mental overhead here. this is explicit and doesn't change that much